spec/lib/parse_spec.rb

```diff
@@ -486,6 +486,101 @@
     )
   end
 
+  it 'correctly finds expressions and function calls in a CREATE INDEX statment' do
+    query = described_class.parse('CREATE INDEX testidx ON test USING btree (a, (b+c), lower(d))')
```

Comment:
I guess what I should really test here is something like `lower(b) || lower(c)`--that was the reason for going via `subselect_items`. Does that make sense?

Reply:
I think this works - since an expression could always contain more sub-expressions, as long as we treat each index item as an expression (i.e. subselect_item) we should be good.